source/slang/slang-emit-cpp.cpp

```diff
@@ -2589,34 +2589,43 @@ void CPPSourceEmitter::emitModuleImpl(IRModule* module)
                             default:
 
                                 auto targetProgram = program->getTargetProgram(targetRequest);
-                                DiagnosticSink sink(linkage->getSourceManager());
+                                //DiagnosticSink sink(linkage->getSourceManager());
```

Comment:
We probably just want to remove this line, as it doesn't seem to add any future value.